src/main/java/Duke.java

```diff
@@ -6,70 +7,90 @@ public static void main(String[] args) {
         );
 
         Scanner sc = new Scanner(System.in);
-        Task[] storage = new Task[100];
+        ArrayList<Task> storage = new ArrayList<Task>();
         int count = 0;
-        String instruction;
+        String instruction; // instruction for the chatbot to follow
         while(true){
             if(sc.hasNextLine()) {
+                // checks if there is another instruction
                 instruction = sc.nextLine();
             }else{
```

Comment:
Small styling violation as, if I am not wrong, there should be a space between the brackets and the condition.
```suggestion
         } else {